framework/wazuh/core/agent.py

```diff
@@ -623,6 +615,7 @@ def reconnect(self, wq: WazuhQueue) -> str:
         str
             Message generated by Wazuh.
         """
+        # TODO: the behavior should be reviewed in the corresponding use case.
```

Comment:
It could be useful to have all TODOs to point to a specific issue so they are easier to find later. For example: `TODO(#123):`. 

It's something I've been doing for the Comms API and I think we could adopt it as a good practice.